language/gpt-3/saxml/README.md

```diff
@@ -0,0 +1,437 @@
+  --log-dir /mlperf_inference/language/gpt-3/saxml/test_evaluation_logs/Offline
+
+```
+##### Run Evaluation with GPT3 Model
+
+###### Server
+
+```
+python evaluation.py \
+  --spm-path ${GPT3_SPM_PATH} \
+  --mlperf-accuracy-file /mlperf_inference/language/gpt-3/saxml/loadgen_logs/Server/accuracy/mlperf_log_accuracy.json \
+  --dataset-path ${CNN_TOKENIZED_DATASET_PATH} \
+  --log-dir /mlperf_inference/language/gpt-3/saxml/evaluation_logs/Server
+
+```
+###### Offline
+
+```
+python evaluation.py \
+  --spm-path ${SPM_PATH} \
+  --mlperf-accuracy-file /mlperf_inference/language/gpt-3/saxml/loadgen_logs/Offline/accuracy/mlperf_log_accuracy.json \
+  --dataset-path ${CNN_TOKENIZED_DATASET_PATH} \
+  --log-dir /mlperf_inference/language/gpt-3/saxml/evaluation_logs/Offline
+```
+
+```
+gsutil cp -r /mlperf_inference/language/gpt-3/saxml/evaluation_logs gs://cnn_dailymail_public/mlperf/evaluation_logs
+
+```
+
+### GPU
+
+GPU_NAME=morgandu-a100
+PROJECT_ID=tpu-prod-env-one-vm
+ZONE=europe-west4-a
+
+##### Create a Google Cloud Bucket
+SAX_ADMIN_STORAGE_BUCKET="sax-admin-mlperf-gp3" # tpu
+SAX_ADMIN_STORAGE_BUCKET="sax-admin-mlperf-gpt-3" # gpu
+gcloud storage buckets create gs://${SAX_ADMIN_STORAGE_BUCKET} --project=${PROJECT_ID}
+
+
+##### Create a Google Cloud GPU
+gcloud compute instances create ${GPU_NAME} \
+   --project ${PROJECT_ID} \
+   --zone ${ZONE} \
+   --machine-type a2-ultragpu-1g \
+   --maintenance-policy TERMINATE \
+   --restart-on-failure \
+   --image-family tf2-ent-latest-gpu \
+   --image-project deeplearning-platform-release \
+   --boot-disk-size 300GB \
+   --metadata "install-nvidia-driver=True,proxy-mode=project_editors" \
+   --scopes https://www.googleapis.com/auth/cloud-platform
+
+gcloud compute ssh ${GPU_NAME} \
+   --project ${PROJECT_ID} \
+   --zone ${ZONE}
+
+
+##### Install Bazel
+
+BAZEL_VERSION=5.4.0
+mkdir ~/bazel
+cd ~/bazel
+curl -fSsL -O https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh
+chmod +x bazel-*.sh
+sudo su
+BAZEL_VERSION=5.4.0
+./bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh
+exit
+
+bazel version
+WARNING: Invoking Bazel in batch mode since it is not invoked from within a workspace (below a directory having a WORKSPACE file).
+Build label: 5.4.0
+Build target: bazel-out/k8-opt/bin/src/main/java/com/google/devtools/build/lib/bazel/BazelServer_deploy.jar
+Build time: Thu Dec 15 16:14:25 2022 (1671120865)
+Build timestamp: 1671120865
+Build timestamp as int: 1671120865
+
+
+###### Build Loadgen
+git clone --recurse-submodules https://github.com/mlcommons/inference.git mlperf_inference
+cd mlperf_inference/loadgen
+CFLAGS="-std=c++14 -O3" python setup.py bdist_wheel
+cd ..; pip install --force-reinstall loadgen/dist/`ls -r loadgen/dist/ | head -n1` ; cd -
+cd ../..
+
+
+###### Reference Implementation DIR
+
+_LOCAL_WORK_DIR="/usr/local/google/home/morgandu/projects/mlcommons-inference/language/gpt-3/saxml"
```

Comment:
Should we generalize to the use the user's id? Or at least write a comment instructing the user to set the `_LOCAL_WORK_DIR` to the `inference/language/gpt-3/saxml` directory within their VM